quickstarts/microsoft.web/webapp-linux-managed-mysql/main.bicep

```diff
@@ -0,0 +1,149 @@
+@description('main.bicep')
+param siteName string = '${uniqueString(resourceGroup().name)}'
+
+@description('Database administrator login name')
+@minLength(1)
+param administratorLogin string
+
+@description('Database administrator password')
+@minLength(8)
+@secure()
+param administratorLoginPassword string
+
+@description('Azure database for mySQL compute capacity in vCores (2,4,8,16,32)')
+@allowed([
+  2
+  4
+  8
+  16
+  32
+])
+param dbSkucapacity int = 2
+
+@description('Azure database for mySQL sku name ')
+@allowed([
+  'GP_Gen5_2'
+  'GP_Gen5_4'
+  'GP_Gen5_8'
+  'GP_Gen5_16'
+  'GP_Gen5_32'
+  'MO_Gen5_2'
+  'MO_Gen5_4'
+  'MO_Gen5_8'
+  'MO_Gen5_16'
+  'MO_Gen5_32'
+])
+param dbSkuName string = 'GP_Gen5_2'
+
+@description('Azure database for mySQL Sku Size ')
+@allowed([
+  102400
+  51200
+])
+param dbSkuSizeMB int = 51200
+
+@description('Azure database for mySQL pricing tier')
+@allowed([
+  'GeneralPurpose'
+  'MemoryOptimized'
+])
+param dbSkuTier string = 'GeneralPurpose'
+
+@description('MySQL version')
+@allowed([
+  '5.6'
+  '5.7'
+])
+param mysqlVersion string = '5.7'
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Azure database for mySQL sku family')
+param databaseskuFamily string = 'Gen5'
+
+var databaseName = '${uniqueString(resourceGroup().id)}'
```

Comment:
in the original sample, these 3 vars (and the siteName param) had static prefixes to make them easier to read - is removing that intentional?  If so, remove the string interpolation on all 3...  If not, add back the prefixes here and on the site name param

Reply:
It was in response to a different comment. I've removed the string interpolation on all three.